webapp/channels/src/components/admin_console/system_roles/system_role/strings.tsx

```diff
@@ -105,7 +105,11 @@ export const sectionStrings: Record<string, Record<string, MessageDescriptor>> =
     user_management_system_roles: defineMessages({
         name: {
             id: 'admin.permissions.sysconsole_section_user_management_system_roles.name',
-            defaultMessage: 'Delegated Granular Administration',
+            defaultMessage: 'System Roles',
```

Comment:
I though that we recently changed the name to `Delegated Granular Administration` on purpose. Was this not the case?

Reply:
Your right. I changed this to match the language file, should have changed the language file. The name in the properties form was never changed. Fixed.